tutorials/cp-portal-cloud-foundry-create-sitelaunchpad/cp-portal-cloud-foundry-create-sitelaunchpad.md

```diff
@@ -22,7 +22,7 @@ In this tutorial we'll create a launchpad site for a recruiting company called `
 
 When you access the SAP Launchpad service, the Site Directory is in focus. From here you'll create your new site.
 
-> Note: in the side panel of the SAP Launchpad service, you'll see four tools. The **Site Directory** where you're going to create a new site. All sites that you create will be displayed here. The **Content Manager** where you'll manage cross-site content such as business apps. The **Provider Manager** where you manage content providers. Content providers expose business content that you can integrate into your launchpad sites. The fourth icon opens **Settings** where you can configure various settings related to the SAP Launchpad service.
+> Note: in the side panel of the SAP Launchpad service, you'll see four tools. The **Site Directory** where you're going to create a new site. All sites that you create will be displayed here. The **Content Manager** where you'll manage cross-site content such as business apps. The **Channel Manager** where you manage different channels that expose business content that you can integrate into your launchpad sites. The fourth icon opens **Settings** where you can configure various settings related to the SAP Launchpad service.
```

Comment:
Please don't label notes with the keyword "Note", since the formatting already makes it a note. If you want to provide more context, you could say something like "Additional Information", etc.
The label should be **bold**.

Also, please start with a capital letter after the colon.